docs/explanation/backend-tables.md

```diff
@@ -69,22 +69,18 @@ the names of the table schemas, tables and columns
 have been provided for you.
 
 For example,
-in the [writing a dataset definition](../tutorial/writing-a-dataset-definition/index.md) section of the tutorial,
-we used the interactive ehrQL sandbox with the following statement to start with:
+in the initial [dataset definition](../tutorial/setting-up/index.md) of the tutorial,
```

Comment:
The dataset first appears in the [working-with-data-with-ehrql](https://docs.opensafely.org/ehrql/tutorial/working-with-data-with-ehrql/) section, not in setting-up

Reply:
I was counting the screenshot into appearances - but I see why that's going to be way more confusing. Thanks!

Reply:
Oh yes - I didn't even notice the dataset definition in the screenshot!